src/tests/unit/keywords-serializer.test.js

```diff
@@ -0,0 +1,29 @@
+import chai from 'chai';
+import proxyquire from 'proxyquire';
+import { testSingleResource, createConfigStub } from './test-helpers';
+
+chai.should();
+
+describe('Test keywords-serializer', () => {
+  const configStub = createConfigStub();
+  const { serializeKeywords } = proxyquire(
+    'serializers/keywords-serializer',
+    {},
+  );
+  configStub.restore();
+
+  it('serializeKeywords should properly serialize keywords collection (POST) in JSON API format', () => {
+    const updatedKeywordCollection = {
+      id: 'fake id',
+    };
+    const result = serializeKeywords(updatedKeywordCollection, { method: 'POST', query: 'fake' });
+    return testSingleResource(result);
+  });
+  it('serializeKeywords should properly serialize keywords collection (GET) in JSON API format', () => {
```

Comment:
In "Test keywords-serializer" and "Test documents-serializer", GET and POST tests are actually exactly the same besides the input method. You should be able to combine them with different var and use template strings to format the testing titles as you did in https://github.com/osu-mist/onbase-docs-api/pull/108/files#diff-c1a7ac1090dae64f3f41dfa948d89eae526e17bdc44df6ae83f4e0e21b471b3cR66